addons/bus/static/tests/bus_connection_alert.test.js

```diff
@@ -31,6 +31,7 @@ test("show warning when bus connection encounters issues", async () => {
         WEBSOCKET_CLOSE_CODES.ABNORMAL_CLOSURE
     );
     await waitForSteps(["reconnecting"]);
+    await runAllTimers();
```

Comment:
Import `CONNECTION_LOST_WARNING_DELAY` and advanceTime(CONNECTION_LOST_WARNING_DELAY`, so the test is more explicit and matches the expected functional behaviour. `runAllTimers()` doesn't mean anything in practice, is it 1s or 2 years?